example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Alexander Matthes, Benjamin Worpitz, Erik Zenker, Matthias Werner
+/* Copyright 2019 Alexander Matthes, Benjamin Worpitz, Erik Zenker, Matthias Werner, Bernhard Manfred Gruber
```

Comment:
Hm, you didn't work for us in 2019. Maybe you can find out the actual date of your commits by using

```
git log --all --after="<date> 00:00" --before="<date> 23:59" --author="<author>"
```

and do this for 2020 and 2021 each? It would be nice to have the year set to "last touched".

Reply:
done